src/python/WMCore/ReqMgr/Service/RequestAdditionalInfo.py

```diff
@@ -6,11 +6,51 @@
 from WMCore.ReqMgr.Utils.Validation import get_request_template_from_type
 
 import WMCore.ReqMgr.Service.RegExp as rx
-from WMCore.REST.Format import JSONFormat
+from WMCore.REST.Format import JSONFormat, PrettyJSONFormat
 
 from WMCore.Wrappers import JsonWrapper
 from WMCore.WMSpec.WMWorkload import WMWorkloadHelper
 
+#TODO: need to get algorithm proper way
+ALGO_DICT = {"FileBased": {"files_per_job": 0,
+                           "include_parents": False},
```

Comment:
I tend to say "include_parents" should not be a splitting changeable parameter. It's defined during creation time with "IncludeParents" and that's it.